_posts/2019-08-13-how-to-deploy-production-grade-vpc-aws.adoc

```diff
@@ -441,7 +441,8 @@ change something in prod rather than pre-prod).
 
 Therefore, your best bet is to put pre-production environments and production environments in completely separate AWS
 accounts. This makes it easy to, for example, grant relatively lax permissions in pre-prod environments, but very
-strict permissions in production. Check out the <<production_grade_aws_account_structure>> guide for instructions.
+strict permissions in production. Check out the  link:../foundations/how-to-configure-production-grade-aws-account-structure[Production Grade AWS Account Structure]
```

Comment:
Is there any version of the links that can automatically pull in the title of the AsciiDoc file or even a subsection of it, so we don't have to copy-paste it?